lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -83,7 +83,7 @@ function runLighthouse(url, configPath, isDebug) {
 
     runCount++;
     console.log(`${log.dim}$ ${command} ${args.join(' ')} ${log.reset}`);
-    runResults = spawnSync(command, args, {encoding: 'utf8', stdio: ['pipe', 'pipe', 'inherit']});
```

Comment:
This seems unnecessary. Also prevents the `DEBUG` printing (few lines later) from working